shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -94,7 +101,20 @@ public int run(CommandLine cl) throws IOException {
    */
   public void generateCapacityReport(GetWorkerReportOptions options, AlluxioConfiguration conf)
       throws IOException {
-    List<WorkerInfo> workerInfoList = mBlockMasterClient.getWorkerReport(options);
+    boolean workerRegisterToAllMasters =
```

Comment:
Interestingly this command may not execute on a node which has this worker config property. But that'd be a super rare case and not worth handling specially.

Reply:
this config has ENFORCE consistency level so this should not happen